samples/app/src/main/kotlin/com/bumble/appyx/app/node/samples/SamplesSelectorNode.kt

```diff
@@ -0,0 +1,159 @@
+package com.bumble.appyx.app.node.samples
+
+import android.os.Parcelable
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.ExperimentalUnitApi
+import androidx.compose.ui.unit.dp
+import com.bumble.appyx.app.node.cards.CardsExampleNode
+import com.bumble.appyx.app.node.slideshow.WhatsAppyxSlideShow
+import com.bumble.appyx.core.composable.ChildRenderer
+import com.bumble.appyx.core.integrationpoint.LocalIntegrationPoint
+import com.bumble.appyx.core.modality.BuildContext
+import com.bumble.appyx.core.navigation.NoOpNavModel
+import com.bumble.appyx.core.node.Node
+import com.bumble.appyx.core.node.ParentNode
+import com.bumble.appyx.core.node.node
+import com.bumble.appyx.navmodel.backstack.BackStack
+import com.bumble.appyx.navmodel.backstack.operation.push
+import com.bumble.appyx.sample.navigtion.compose.ComposeNavigationRoot
+import kotlinx.parcelize.Parcelize
+
+class SamplesSelectorNode(
+    buildContext: BuildContext,
+    private val samplesContainerBackStack: BackStack<SamplesContainerNode.NavTarget>
```

Comment:
children have dependency on the parent, this is bad practice, they should produce an output instead

Reply:
can do :)

Reply:
@KovalevAndrey done. though should I use a NodeConnector? It doesn't necessarily need to be done with Rx, and I'm not quite sure what our approach is when we don't use Rx